<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QWidget#frame{
background-color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>10</y>
         <width>421</width>
         <height>781</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#widget{
background-color: rgb(255, 255, 255);
border-radius:10px;
}</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 87 35pt &quot;Nexa Heavy&quot;;
color:rgb(85, 0, 127);</string>
        </property>
        <property name="text">
         <string>Bienvenido!</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 87 12pt &quot;Nexa Heavy&quot;;
color: grey;</string>
        </property>
        <property name="text">
         <string>Ingresa con tus credenciales</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="InputEmail">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>300</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: rgb(243, 243, 243);
    border-radius: 5px;
    font: 87 12pt &quot;Nexa Heavy&quot;;
    border: 2px solid rgb(85, 0, 127);
}

QLineEdit:focus {
    border: 0px solid white;
    background-color: rgb(255, 255, 255);
    border-bottom: 2px solid rgb(145, 67, 255);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string notr="true">Ingrese un email</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="InputConstra">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>410</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color: rgb(243, 243, 243);
	border-radius:5px;
	font: 87 12pt &quot;Nexa Heavy&quot;;
	border:2px solid rgb(85, 0, 127);
}

QLineEdit:focus{
	
	border: 0px solid white;
	
	background-color: rgb(255, 255, 255);
	border-bottom:2px solid rgb(145, 67, 255);
}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Ingrese su contrasena</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>260</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 87 15pt &quot;Nexa Heavy&quot;;
color:rgb(85, 0, 127);</string>
        </property>
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>370</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 87 15pt &quot;Nexa Heavy&quot;;
color:rgb(85, 0, 127);</string>
        </property>
        <property name="text">
         <string>Constrasena:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnIngresar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>560</y>
          <width>311</width>
          <height>41</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{font: 87 15pt &quot;Nexa Heavy&quot;;
color:rgb(255, 255, 255);
border-radius:10px;
background-color: rgb(85, 0, 127);
}
QPushButton:hover{
	
	
	background-color: rgb(145, 67, 255);
}</string>
        </property>
        <property name="text">
         <string>Ingresar</string>
        </property>
       </widget>
       <widget class="QLabel" name="ErrorEmail">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>330</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 87 12pt &quot;Nexa Heavy&quot;;
color: red;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="ErrorConstra">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>450</y>
          <width>251</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 87 12pt &quot;Nexa Heavy&quot;;
color: red;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>-90</x>
         <y>0</y>
         <width>1071</width>
         <height>791</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/4/HD-wallpaper-dark-purple-windows-11-logo-windows-11.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_5</zorder>
      <zorder>widget</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="1.qrc"/>
 </resources>
 <connections/>
</ui>
